tests/js/test.js

```diff
@@ -0,0 +1,70 @@
+/*
+
+A test script to look at how Nightmare behaves natively.  This should not be used.
```

Comment:
"Now for your own safety and the safety of those around you, metallic objects must never be placed within the vault"